src/pytest_dbfixtures/factories/postgresql.py

```diff
@@ -121,7 +121,7 @@ def drop_postgresql_database(psycopg2, user, host, port, db):
     conn = psycopg2.connect(user=user, host=host, port=port)
     conn.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
     cur = conn.cursor()
-    cur.execute('DROP DATABASE IF EXISTS %s' % db)
+    cur.execute('DROP DATABASE IF EXISTS %s;' % db)
```

Comment:
yes, we haven't had any problems with this except for errors we get when some process used in integration tests doesn't close the connection at the end of the test.

I'm still going to accept this for the sake of correspondence with other tools (better to add comma after SQL statement here, than forget it about in other places.

Still even documentation doesn't always use the semicolon for single statement execute: http://initd.org/psycopg/docs/usage.html

It could help to get to the issue, if you'd write something about your test setup or test case.